src/Tribe/Page.php

```diff
@@ -84,7 +84,17 @@ public function get_menu_hook() {
 	 * @since 1.0.0
 	 */
 	public function add_menu() {
-		$parent = class_exists( 'Tribe__Events__Main' ) ? Tribe__Settings::$parent_page : Tribe__Settings::$parent_slug;
+		if ( class_exists( 'Tribe__Events__Main' ) ) {
```

Comment:
This conditional isn't needed - the plugin won't run w/o TEC:
https://github.com/mt-support/tribe-ext-test-data-generator/blob/main/src/Tribe/Plugin_Register.php#L29